compiler/src/typed/dependency_graph.re

```diff
@@ -106,45 +106,27 @@ module Make = (DV: Dependency_value) => {
     do_register(dependency);
   };
 
-  let solve_next_out_of_date = (~stop=?, ()) => {
-    let (stop_found, ret) =
+  let get_dependencies = () => {
+    List.rev(
       G_topological.fold(
-        ((dep, state), acc) => {
-          switch (acc) {
-          | (true, _) => acc
-          | (false, Some(_)) => acc
-          | (false, None) =>
-            let stop_found =
-              switch (stop) {
-              | Some(d) when DV.equal(d, dep) => true
-              | _ => false
-              };
```

Comment:
I assume the code you removed here was due to the `stop_found` not being used but is it not important?

Reply:
It was necessary before because it would iterate to find the next dependency that was out of date so it could be worked on. Now, we just grab all of them at once.